.github/workflows/main.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        rust: [stable, beta, nightly]
+        rust: [stable, beta, nightly, 1.31.0, 1.32.0, 1.33.0, 1.34.0, 1.35.0, 1.36.0, 1.37.0]
```

Comment:
Mind removing these older verisons? I personally prefer to just test with stable/beta/nightly nowadays

Reply:
Done.